karma.conf.js

```diff
@@ -124,7 +124,7 @@ module.exports = function(config) {
     // possible values: 'dots', 'progress'
     // available reporters: https://npmjs.org/browse/keyword/karma-reporter
     reporters: ['coverage', 'spec'],
-    specReporter: {showSpecTiming: true, suppressSkipped: true},
+    specReporter: {showSpecTiming: true, suppressPassed: true},
```

Comment:
Why do you suppress passed here? Having the time it took to execute is a  nice feature to help detect slow tests

Reply:
`suppressSkipped` should also stay. Otherwise we will have a lot of these statements when making use of `fit` or `fdescribe`:

![image](https://user-images.githubusercontent.com/469989/51265051-1ba53700-19b8-11e9-96db-0070dad422a3.png)